packages/amplify-nodejs-function-template-provider/resources/lambda/trigger/trigger-dynamodb.js.ejs

```diff
@@ -5,7 +5,7 @@
  */
 exports.handler = event => {
   console.log(`EVENT: ${JSON.stringify(event)}`);
-  event.Records.forEach(record => {
+  for (const record of event.Records) {
```

Comment:
There's a closing `)` that needs to be removed too.

Reply:
ah good call, removed